README.md

```diff
@@ -9,124 +9,136 @@ The following repository contains:
+```
 
-   The script will look for open-cv, numpy, imutils, ximer driver only after successfully installing python! So if you don't have python it won't print missing python dependent packages except python itself.
+3. Pull frontend image from GCR
 
-   ```bash
-   cd SmartKickers
-   ./requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
+```
 
-   > **NOTE:** Be sure to give permission for the script to execute.<br/>
+4. Pull backend image from GCR
 
-   ```bash
-   sudo chmod +x requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
+```
 
-3. Install missing dependencies.
+## Launching
 
-4. In folder [`SmartKickers`](SmartKickers) run docker-compose build.
+> **NOTE:** Configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
 
-   ```bash
-   docker-compose build
-   ```
+### Launching the server and web application
 
-## Launching
+- Run pulled containers
+
+  - In a terminal run:
 
-> **NOTE:** Configuration files for the camera view are provided by the developers in [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
+  ```bash
+  docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
+  ```
+
+  - Open a new terminal and run:
 
-1. In the folder [`SmartKickers`](SmartKickers) launch the docker-compose.
+  ```bash
+  docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
+  ```
 
-   ```bash
-   docker-compose up
-   ```
+### Launching the ball-tracking software
 
-   After launching you will see that the backend and the frontend of the application has started successfully.
+1. Plug in the camera to the computer.
 
-   <h1 align="center">
-       <img src="assets/appLaunch.png">
-   </h1>
+2. In a new terminal window, launch the Node.js server.
 
-2. Plug-in the camera wire to the computer.
+In order to launch the application, navigate to the `SmartKickersAI` repo and run:
 
-3. In new terminal switch to [`SmartKickersAI/LocalServer/server`](SmartKickersAI/LocalServer/server) and launch the node server.
+```bash
+cd LocalServer/server
+node server.js
+```
 
-   ```bash
-   cd SmartKickersAI/LocalServer/server
-   node server.js
-   ```
+Successful connection to the Go server shows the connection details like in the screenshot below:
 
-   You will see the following output indicating that we have successfully connected to the go server.
+![Node launch image](assets/nodeLaunch.png "Node launch")
 
-   <h1 align="center">
-       <img src="assets/nodeLaunch.png">
-   </h1>
+3. In your Internet browser, go to [`localhost:3007`](http://localhost:3007/) to see the React application.
 
-3. In the browser, go to [`localhost:3007`](http://localhost:3007/) to see the react application.
+This takes you to the SmartKickers React application.
```

Comment:
You already added it to the previous sentence, so this sentence doesn't bring in anything new to the table. 
But it's good that you're ending the instructions with the expected result. 
I propose that you change it to something like: 
```suggestion
You can now play the SmartKickers game. Enjoy!
```